tests/Integration/Notifications/SendingMailNotificationsTest.php

```diff
@@ -143,7 +143,7 @@ public function test_mail_is_sent_to_named_address()
 
         $user->notify($notification);
     }
-
+    
```

Comment:
Just nitpicking but this 4 spaces should not be here.

Reply:
Sure 👍